checkov/main.py

```diff
@@ -415,6 +425,40 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
     return None
 
 
+def report_contributor_metrics(repository: str) -> None:  # ignore: type
+
+    def _parse_gitlog() -> dict[str, Any] | None:
+        process = subprocess.Popen(['git', 'shortlog', '-ne', '--all', '--since', '"90 days ago"', '--pretty="commit-%ct"', '--reverse'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
```

Comment:
@maxamel Can we use gitpython instead of subprocess?
https://pypi.org/project/GitPython/

Reply:
I also wanted to suggest it, but it looks like that `shortlog` doesn't work very well with `gitpython` https://github.com/gitpython-developers/GitPython/discussions/1434